testsuite/integration/basic/src/test/java/org/wildfly/test/integration/microprofile/metrics/metadata/MicroProfileMetricsMetadataTestCase.java

```diff
@@ -110,10 +113,10 @@ public void testJvmUptimeMetric() throws Exception {
     }
 
     private void checkJvmUptime(JsonObject jvmUptime) {
-        assertEquals(jvmUptime.getString("type"), "gauge");
-        assertEquals(jvmUptime.getString("unit"), "milliseconds");
-        assertEquals(jvmUptime.getString("displayName"), "JVM Uptime");
-        assertEquals(jvmUptime.getString("description"), "Displays the uptime of the Java virtual machine");
+        assertEquals("gauge", jvmUptime.getString("type"));
+        assertEquals("milliseconds", jvmUptime.getString("unit"));
+        assertEquals("JVM Uptime", jvmUptime.getString("displayName"));
+        assertEquals("Displays the uptime of the Java virtual machine", jvmUptime.getString("description"));
```

Comment:
Seems to come from base-metrics.properties. Are those meant to be localised? If they are this test might fail in the future. It isn't introduced by this PR though, so it is probably fine

Reply:
I don't know if they will ever be localized (the consumers of these metrics are other systems such as Prometheus)  In any case, I think that's out of scope of this upgrade.